renderer/components/__tests__/Sidebar.test.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import React from 'react'
+import { screen, render, cleanup, fireEvent } from '@testing-library/react'
+import { theme } from 'ooni-components'
+import { ThemeProvider } from 'styled-components'
+import { IntlProvider } from 'react-intl'
+import English from '../../../lang/en.json'
+
+import Sidebar, { NavItem, navigationPaths } from '../Sidebar'
+import { version } from '../../../package.json'
+
+// Mocking useRouter()
+const mockRouterPush = jest.fn()
+jest.mock('next/router', () => ({
+  useRouter() {
+    return {
+      route: '/',
+      pathname: '/dashboard',
+      push: mockRouterPush
+    }
+  },
+}))
+
+const renderComponent = (component, locale = 'en', messages = English) => {
+  return render(
+    <IntlProvider locale={locale} messages={messages}>
+      <ThemeProvider theme={theme}>{component}</ThemeProvider>
+    </IntlProvider>
+  )
+}
+
+describe('Tests for Sidebar component', () => {
+  afterEach(() => {
+    cleanup()
+  })
+
+  test('Displays the correct version', async () => {
+    renderComponent(
+      <Sidebar>
+        <></>
+      </Sidebar>
+    )
+    expect(screen.getByText(version)).toBeInTheDocument()
+  })
+
+  test('Displays child elements', async () => {
+    renderComponent(
+      <Sidebar>
+        <p>Test paragraph</p>
+      </Sidebar>
+    )
+    expect(screen.getByText('Test paragraph')).toBeInTheDocument()
+  })
+
+  test('NavItem renders Navigation List correctly in the Sidebar', async () => {
```

Comment:
Can we not directly test by rendering `<Sidebar>` and checking if it has all the items in navigationPath and if each of them navigates to the right page?